### ISSUE TITLE
RNN to model  lowpass filter

### ISSUE THREAD

**Issue by PeterPirog** [keras-team/keras#15967]
I would like to simulate lowpas filter IIR [Infinite impulse response](https://en.wikipedia.org/wiki/Infinite_impulse_response) with high time delay.

I know input values X(n), X(n-1), X(n-2)  ...   and  output valuesY(n), Y(n-1), Y(n-2) because I recorded it's from physical device

`Y(n)=a1 x Y(n-1)+a2 x Y(n-2)+...+ak  x Y(n-k)+b0 x X(n)+b1 x Y(n-1)+...+bk x X(n-k)`

X and Y are one dimensional, I will be grateful for suggestions what is the best order of recurent layers? 

What is the best way to split data for training and validation set? Maybe someone solved similar problem.

Practically, I would like to make RL controler for temperature bath and I need to prepare simulator at first. X values are {0 - heater off, 1 - heater on}, Y values are measured temperatures.
Delay is the time between turning on/off the heater and  changing the measured temperature (delay is about 70 second, sampling 5 s).

I found something [similar ](http://www.ijeee.net/uploadfile/2014/0711/20140711044156242.pdf) but I need to use IIR filter not FIR, and I thing RNN can be beter than only dense layers.

**Comment by haifeng-jin** [comment]
@PeterPirog Thanks for the issue.
I suggest you to go [these places](https://keras.io/about/#support) for help.
Here is only for bug reports and feature requests.
